src/main/resources/db/migration/V1_38__document_simpler_tables.sql

```diff
@@ -0,0 +1,29 @@
+COMMENT ON TABLE auth_user IS 'details about the user from hmpps-auth';
+COMMENT ON COLUMN auth_user.id IS 'the user ID';
+COMMENT ON COLUMN auth_user.auth_source IS 'where the user has come from';
```

Comment:
i think this description is lacking in clarity, but explaining this field in a few words is tricky

Reply:
would be great to iterate on it!